pim-tool-front/css/styles.css

```diff
@@ -0,0 +1,25 @@
+/* You can add global styles to this file, and also import other style files */
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+:root {
+  --blue: #2f85fa;
```

Comment:
[N] Create css variables and reuse the variable
https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties